packages/resizable-mixin/resizable-mixin.ts

```diff
@@ -0,0 +1,52 @@
+import { LitElement } from 'lit-element';
+import getResizeObserver from './resize-observer-polyfill';
+import { ResizableClass } from './resizable-class';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T = object> = new (...args: any[]) => T;
+
+export const ResizableMixin = <T extends Constructor<LitElement>>(
+  base: T
+): T & Constructor<ResizableClass> => {
+  class Resizable extends base {
+    private _resizeObserver?: ResizeObserver;
```

Comment:
Would it make sense to only have 1 (static) instance of the ResizeObserver?
Something like:

```js
new ResizeObserver(entries) => {
  entries.forEach(entry => entry.target._sizeChanged(entry.contentRect));
});
```

Reply:
Thanks, done.